libs/mimir/src/adapters/primary/common/dsl.rs

```diff
@@ -121,10 +121,15 @@ fn build_boosts(
         decay.offset = proximity.offset;
         decay.decay = proximity.decay;
     }
+    let weight_boost = match query_type {
+        QueryType::PREFIX => settings.importance_query.proximity.weight,
+        _ => settings.importance_query.proximity.weight_fuzzy,
```

Comment:
```suggestion
        QueryType::FUZZY => settings.importance_query.proximity.weight_fuzzy,
```
So we won't forget about this if a new QueryType is added

Reply:
:+1: